service/arangod_config_builder.go

```diff
@@ -165,6 +166,11 @@ func createArangodArgs(log zerolog.Logger, config Config, clusterConfig ClusterC
 		optionPair{"--log.file", slasher(myContainerLogFile)},
 		optionPair{"--log.force-direct", "false"},
 	)
+
+	if !config.RunningInDocker && features.CopyInstallationFiles() {
```

Comment:
Is this needed for all server types in all deployment types?

Reply:
Good point, will adjust

Reply:
ok technically its a bit complicated: on 3.4 only single-servers / coordinators need it, but on 3.3.17 all servers will need it. Since 3.4 will just ignore the option on dbservers/ agents I think we can just leave it like that